src/screens/Feed/FeedScreen.tsx

```diff
@@ -35,10 +35,10 @@ const FeedScreen = ({ navigation }: any) => {
           alignItems: 'center',
           width: '100%'
         }}>
-        {list.map((job) => {
+        {list.map((job, index) => {
```

Comment:
what's this index used for

Reply:
it's so that the eslint won't get mad at us for not having a unique index prop